chat-client/src/client/mynahUi.test.ts

```diff
@@ -0,0 +1,153 @@
+import { before, afterEach } from 'mocha'
+import sinon = require('sinon')
+import { assert } from 'sinon'
+import { createMynahUi, InboundChatApi, handleChatPrompt } from './mynahUi'
+import { Messager, OutboundChatApi } from './messager'
+import { TabFactory } from './tabs/tabFactory'
+import { ChatItemType, MynahUI } from '@aws/mynah-ui'
+
+describe('MynahUI', () => {
+    let messager: Messager
+    let mynahUi: MynahUI
+    let api: InboundChatApi
+    let chatApi: OutboundChatApi
```

Comment:
nit: I would keep names consistent with types here, otherwise it's confusing what api does what.